packages/dm-core-plugins/src/job/JobControlButton.tsx

```diff
@@ -1,53 +1,57 @@
 import { Button, CircularProgress, Icon } from '@equinor/eds-core-react'
-import { play, stop } from '@equinor/eds-icons'
-import { JobStatus, StartJobResponse } from '@development-framework/dm-core'
+import { play, stop, refresh } from '@equinor/eds-icons'
+import { JobStatus } from '@development-framework/dm-core'
 import React, { MutableRefObject, useRef, useState } from 'react'
 
 export const JobControlButton = (props: {
   jobStatus: JobStatus
   jobExists: boolean
   createJob: () => Promise<unknown>
-  start: () => Promise<StartJobResponse | null>
-  halt: () => void
+  start: () => void
 }) => {
-  const { jobStatus, start, halt, jobExists, createJob } = props
+  const { jobStatus, start, jobExists, createJob } = props
   const [hovering, setHovering] = useState(false)
   const buttonRef: MutableRefObject<HTMLButtonElement | undefined> = useRef()
   buttonRef.current?.addEventListener('mouseenter', () => setHovering(true))
   buttonRef.current?.addEventListener('mouseleave', () => setHovering(false))
 
-  function handleClick(): void {
-    if (jobStatus === JobStatus.Running) halt()
-    else {
-      if (!jobExists) {
-        createJob().then(() => {
-          start()
-        })
-      } else {
-        start()
-      }
-    }
+  function handleClick() {
+    if (!jobExists) return createJob().then(() => start())
```

Comment:
`createJob` allready handles `jobExists`, and either updates or creates new. Right now it is not possible to update the job if it already exist. handleClick should just call `createJob` then `start`